app/exporters/formatters/manual_section_indexable_formatter.rb

```diff
@@ -1,4 +1,6 @@
-class ManualSectionIndexableFormatter
+require "formatters/abstract_indexable_formatter"
+
+class ManualSectionIndexableFormatter < AbstractIndexableFormatter
   def initialize(section, manual)
     @section = section
```

Comment:
No good reason, no - I think I noticed the `attr_reader :entity` later on and added the note in the commit message, and didn't consider using it instead of `section`. Done now :)